### ISSUE TITLE
Strange behaviour with tables

### ISSUE THREAD

**Issue by mxwli** [kuzudb/kuzu#2775]
The following queries cause some weird behaviour with kuzu
```
create node table Person(id serial, primary key(id));
create rel table Knows(FROM Person TO Person);
create (:Person {id: 0});
```
From the last query, I get a strange error message: `Error: Cannot write to file. path: tmpdatabase/data.kz fileDescriptor: 7 offsetToWrite: 17592186040320 numBytesToWrite: 4096 numBytesWritten: 18446744073709551615. Error: File too large`
When dropping the table:
```
drop table Knows;
```
Kuzu hangs. I've tried this both on my local computer and on the servers.

**Comment by semihsalihoglu-uw** [comment]
We should not allow specifying the serial property anyways, but let's also find out what's happening with the bug and make sure we don't get the system to states that hang.

**Comment by acquamarin** [comment]
```
kuzu> create node table Person(id serial, primary key(id));
----------------------------------------
| result                               |
----------------------------------------
| Node table: Person has been created. |
----------------------------------------
(1 tuple)
Time: 4.53ms (compiling), 1.63ms (executing)
kuzu> 
kuzu> create rel table Knows(FROM Person TO Person);
--------------------------------------
| result                             |
--------------------------------------
| Rel table: Knows has been created. |
--------------------------------------
(1 tuple)
Time: 0.43ms (compiling), 2.09ms (executing)
kuzu> create (:Person {id: 0});
-
(0 tuples)
Time: 5.93ms (compiling), 1.14ms (executing)
kuzu> match (p:Person) return *;
-------------------------------------
| p                                 |
-------------------------------------
| {_ID: 0:0, _LABEL: Person, id: 0} |
-------------------------------------
(1 tuple)
Time: 4.66ms (compiling), 0.65ms (executing)
kuzu> 
```
I am not able to reproduce the bug, which version are you using?

**Comment by mxwli** [comment]
I'm using the latest release: 0.1.0

**Comment by mxwli** [comment]
I've tested these same queries with the latest build and they seem to work just fine. Seems like the bug was fixed at some point after 0.1.0. Closing the issue.
